en/contribute/dev_call.md

```diff
@@ -1,13 +1,11 @@
-# Weekly Dev Call
+# Weekly Community Q&A Call
```

Comment:
So there is no dev call?

Reply:
Ah, answered in the description of the PR

Reply:
```suggestion
# Weekly Community Q&A Call (Previously "Dev Call")
```